app/Models/InstanceSettings.php

```
@@ -42,6 +42,11 @@ protected static function booted(): void
                     }
                 });
             }
+
+            // Clear trusted hosts cache when FQDN changes
+            if ($settings->isDirty('fqdn')) {
+                \Cache::forget('instance_settings_fqdn_host');
```

PHPMD (2.15.0)

Avoid using static access to class '\Cache' in method 'booted'. (undefined)

(StaticAccess)